game/scripts/npc/abilities/nevermore_dark_lord.txt

```diff
@@ -28,13 +28,13 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "presence_armor_reduction"                        "-4 -5 -6 -7 -10 -17"
+        "presence_armor_reduction"                        "-4 -5 -6 -7 -10 -15"
```

Comment:
this should be -3/-4/-5/-6/-9/-12

Reply:
and then talent should be -12

Reply:
so from a total of -36 to -24

Reply:
I don't want to kill a hero. So no.

Reply:
how does that kill a hero? He literally is still a top 3 dmg output hero with the nerfs